Makefile

```diff
@@ -403,9 +403,9 @@ KBUILD_CXXFLAGS   := -c -g -std=c++11 \
 
 KBUILD_AFLAGS   := -c -g -xassembler-with-cpp -D_ASMLANGUAGE
 
-LDFLAGS += $(call ld-option,-nostartfiles)
+LDFLAGS += $(if $(filter $(ARCH), mips),,$(call ld-option,-nostartfiles))
```

Comment:
Could there be an explanation here in english explaining what is special about mips wrt. -nostartfiles ?

Reply:
Currently this port uses `_init` and `_fini`, I realise you would like this to be more self contained, we are looking into how best to do this.